### ISSUE TITLE
ISO building fails after building Kernel 

### ISSUE THREAD

**Issue by tajummal-lab** [Augustin-FL/building-pfsense-iso-from-source#9]
Dear Augustin,

i`m facing one issue during building ISO after building kernel successfully ,
the screenshot of the error is mention below.
please find the attachment below.

regards: 
![iso_building_failed](https://user-images.githubusercontent.com/56956553/68359341-648fcd00-013d-11ea-8aef-2e59b19bd631.PNG)
Tajummal

**Comment by Augustin-FL** [comment]
Hi Tajummal
there is no indication on the error that happened 

please check what has actually triggered the error

also, for information, issue trackers are here for reporting bugs, not for asking for help.

**Comment by tajummal-lab** [comment]
sorry for that actually i`m not able to find the actual problem there is no
indication or not logged.
what is yours observation? what you think

**Comment by Augustin-FL** [comment]
I think that you should remove the set -x and try again, the error will be much clearer to you

**Comment by Augustin-FL** [comment]
did you find the error?

may I close this issue?

**Comment by tajummal-lab** [comment]
Hi sir,

Actually I don't try today because I was busy in some other task.

So Augustin, kindly cooperate with me.
It's veey tough time for me.
And i think some other person seerekumar facing the same problem

**Comment by tajummal-lab** [comment]
![error](https://user-images.githubusercontent.com/56956553/68745601-c4441780-0618-11ea-81d3-190051f0a15a.PNG)
sorry for late reply
actually the error was this
unable to access
file/root/pfsense/tmp/stage-dir/usr/local/share/libreSense/fix_for_SA-19-10.ufs:No
such file or directory

Regards: Tajummal

**Comment by Augustin-FL** [comment]
hello,

I have updated the guidelines few days ago regarding this issue:

```
cd /root/pfsense

# PKG signing key
mv src/usr/local/share/pfSense src/usr/local/share/${product_name}
rm src/usr/local/share/${product_name}/keys/pkg/trusted/*
cp /root/sign/fingerprint src/usr/local/share/${product_name}/keys/pkg/trusted/fingerprint
``` 
could you ensure that folder pfSense has been renamed to your own product name in `src/usr/local/share/`?

**Comment by tajummal-lab** [comment]
I think these above commands are same as you mentioned in doc.
And when we need to check the renamed folder before changing above commands
or after??

**Comment by tajummal-lab** [comment]
Tell me i thing.
After updating above commands we have to follow all steps again i means
(can i need to delete jail then re create and build ports again??)
You know well that creating jail and ports building process is very
painfull😄😄

**Comment by Augustin-FL** [comment]
i would advice you to run ```/root/build.sh --clean-builder``` then ```/root/build.sh --skip-final-rsync iso``` again?

**Comment by tajummal-lab** [comment]
hello
receiving the same error as above
but folder is created

**Comment by sreekupm** [comment]
Dear Tajummal,

Please share your error.

**Comment by Augustin-FL** [comment]
Closing because fixed in bb1a84d9e81f1accd911860d731d25dee07329f2
